_licenses/cern-ohl-p-2.0.txt

```diff
@@ -0,0 +1,217 @@
+---
+title: CERN Open Hardware Licence Version 2 - Permissive
+spdx-id: CERN-OHL-P-2.0
+nickname: CERN OHL v2 Permissive
+
+description: The CERN OHL was drafted with hardware in mind.  On top of licensing copyright like software and documentation licenses, it also licenses patents.  This “Permissive” variant allows people to take your code, relicense it and use it without any obligation to distribute the sources when they ship a product; however, it does have an attribution requirement.
```

Comment:
I agree generally, but I would say "without sources" instead of "without source code" at the end. Not all designs are made of code. "Source" OTOH is a term defined in the licence text.